heroic-core/src/main/proto/spotify_100.proto

```diff
@@ -20,8 +20,24 @@ message Metric {
 
   // resource "tags" associated to metric.
   map<string, string> resource = 5;
+
+  // Distribution type value
+  Value distribution_type_value = 6;
+}
+
+message Message {
```

Comment:
I just want to point out - this Message object is never used in either FFWD or here. 
1. Maybe we shouldn't add this here as it is not used?
2. And maybe we should remove it from FFWD proto definition to make it identical?
Thoughts?

Reply:
Done